Gruntfile.js

```diff
@@ -56,7 +57,10 @@ module.exports = function(grunt) {
         }
       },
       e2e: {
-        src: ['tests/e2e/**/*.js'],
+        src: [
+          'tests/e2e/*.spec.js',
+          'tests/e2e/**/*.js',
```

Comment:
In https://github.com/medic/medic-api/tree/master/tests/e2e, there are utility classes which will match `*.js`.

`.spec.` is meant to signify that the file in question is a test file.  `.spec.js` is a fairly common convention for test file names.  [In the mocha docs](https://mochajs.org/#browser-specific-methods) they use `test.whatever.js`.  I don't mind if we drop it, and files without `describe()` calls are probably ignored by the test runner, but it seems useful to me to have some prefix or suffix convention to denote files which contain tests rather than just have test-related code in them.